Forces AMGX to use external dependencies, avoiding needing the git submodule
the entire (deprecated) thrust library and all its dependencies.
author: micketeer@gmail.com
--- CMakeLists.txt.orig	2024-03-30 01:54:34.469780980 +0100
+++ CMakeLists.txt	2024-03-30 01:54:46.491884432 +0100
@@ -251,7 +251,6 @@
 ENDIF()
 
 # We depend on a specific version of thrust now so include the submodule
-add_subdirectory("thrust")
 find_package(Thrust REQUIRED CONFIG)
 thrust_create_target(Thrust)
 
